src/Sentry.Unity.Editor/Batch.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using UnityEditor;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor
+{
+    public static class Batch
```

Comment:
This class intruments the "Tools -> Sentry" integration we also refer to as "Sentry Editor Window".
So an alternative name could be:

```suggestion
    public static class SentryEditorWindowInstrumentation
```